app/controllers/organizations/invitations_controller.rb

```diff
@@ -29,4 +29,8 @@ def user_params
       .permit(:first_name, :last_name, :email,
         staff_account_attributes: [:roles])
   end
+
```

Comment:
Nice. So this method is overriding the same method in `DeviseInvitable::Controllers::Helpers` ?

Reply:
Yes. This method override the same method existing in Devise Invitable gem. I found method name "after_sign_out_path_for(resource_name)" in Devise Invitable documentation.

Reply:
Ah nice find!